doc/nrf/libraries/modem/nrf_modem_lib.rst

```diff
@@ -329,6 +329,76 @@ Other configurations set in the devicetree, such as the current speed, are overw
 
    When one of the UART trace backends is enabled by either the Kconfig option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART` or :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_SYNC`, it initializes the UART1 driver, regardless of its status in the devicetree.
 
+Modem trace shell command
+==========================
+
+Shell command: ``modem_trace``
+
+Use the modem trace commands to control the trace functionality in the modem when :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_SHELL` is enabled.
+
+A trace backend that can store modem traces is required to upload modem traces to the cloud.
+
+Examples
+--------
+
+* Modem trace everything (LTE, IP and GNSS):
+
+  .. code-block:: console
+
+     modem_trace start full
+     <test using gnss-, lte-, or ip-commands>
```

Comment:
References to `modem_shell` commands should probably be removed from here.

Reply:
Good catch 👍